### PR TITLE
Fix creation date field

### PR DESCRIPTION
**Pull request by miguel-domatix**
I have fixed the create_date field to be read only.

### CODE REVIEW

**Review by jarroyomorales — APPROVED** [review]

File: helpdesk_mgmt/views/helpdesk_ticket_view.xml [hunk 1]
```diff
@@ -80,7 +80,7 @@
 
                             <field name="priority" widget="priority"/>
                             <field name="company_id" options="{'no_create': True}" groups="base.group_multi_company"/>
-                            <field name="create_date"/>
+                            <field name="create_date" attrs="{'readonly': True}"/>
```
Comment (pedrobaeza) [inline helpdesk_mgmt/views/helpdesk_ticket_view.xml]:
```suggestion
                            <field name="create_date" readonly="1"/>
```

File: helpdesk_mgmt/views/helpdesk_ticket_view.xml [hunk 2]
```diff
@@ -158,7 +158,7 @@
                 <field name="user_id"/>
                 <field name="stage_id"/>
                 <field name="priority" widget="priority"/>
-                <field name="create_date"/>
+                <field name="create_date" attrs="{'readonly': True}"/>
```
Comment (pedrobaeza) [inline helpdesk_mgmt/views/helpdesk_ticket_view.xml]:
```suggestion
                <field name="create_date" readonly="1"/>
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I have squashed on merge, although I don't like this as the bot has not changed the version number. Please forward-port the patch in one commit for v12.

**Comment by miguel-domatix** [comment]
Pedro, I don't understand the commit with that forward-port.

**Comment by pedrobaeza** [comment]
I mean that propose a PR for v12 with only one commit.

**Comment by miguel-domatix** [comment]
Perfect, as soon as possible I will do it.

**Pull request merged**